application/dash_application/dash_multipage_index.py

```diff
@@ -43,6 +57,11 @@ def display_page(pathname):
         else:
             return index_page
 
+    ## As an example of ids being local to each page, this callback
+    ## outputs to dash_multipage_2's 'page-content', which is distinct
+    ## from this page's 'page-content'. This callback drives the 
+    ## interactivity in dash_multipage_2 despite being in the
+    ## dash_multipage_index file.
```

Comment:
I like this. I would move it above the `init_callbacks` function to really reinforce that callbacks ids are in the global space.